### DISCUSSION TITLE
32 bit version of ODBC driver for Windows

### DISCUSSION THREAD

**Discussion by rodryquintero** [duckdb/duckdb#12744] [Ideas]
32 bit applications do not have access to 64 bit ODBC drivers (e.g. Microsoft Report Builder). This will expand Duckdb's use cases as there should be more 32 bit applications experiencing this issue.

## Links
- [64 bit version of SQL Server Report Builder](https://stackoverflow.com/questions/23267052/64-bit-version-of-sql-server-report-builder)
- [Microsoft Report Builder](https://www.microsoft.com/en-us/download/details.aspx?id=53613)

**Comment by nRoger-Env** [comment]
I would like to add that it would help migrate to duckdb easier.
I am stuck with a 32 bits Access database that I would like to migrate to duckDB. That mean that my app is currently 32 bits. Since duckDB doesn'T seems to support 32 bits, I can't incrementally add Duckdb to my app. I would need to convert it all in one go to duckdb while at the same time upgrading to 64 bits.
